dart/lib/vm_service_lib.dart

```diff
@@ -221,6 +879,7 @@ class VmService {
   /// Note that breakpoints are added and removed on a per-isolate basis.
   ///
   /// See [Breakpoint].
+  @override
```

Comment:
I think we should remove the dartdocs for these override methods - the parent methods have good docs already.

Reply:
seems reasonable, done